### ISSUE TITLE
MQTT energy meter - data usage

### ISSUE THREAD

**Issue by home409ca** [xyphro/Sun2000MeterTransposer#2]
I have a Supla MEW-01 meter that sends all measurement data via MQTT to a local broker.

Is it possible to use this data so that the ESP8266 connects to this broker, downloads the data and forwards it to the inverter?

I cannot connect another meter (no space in the switchboard).

Thanks

**Comment by xyphro** [comment]
In case of Supla MEW-01 which has Wifi it might be easier to connect the Sun2000 with a USB to MQTT adapter to a raspberry PI and make a python script there to handle the Sun2000 requests.

**Comment by xyphro** [comment]
I just see i even supports MQTT. This would make it very easy with a bit of python magic

**Comment by home409ca** [comment]
How to do it?
I'm not a programmer...

**Comment by xyphro** [comment]
I agree, that makes it difficult. But maybe it is a good opportunity to learn it. Python makes things easy to beginners.
